vector/v.random/testsuite/test_v_random.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+output = 'test01'
+output2 = 'test02'
+npoints = 100
+state = 'boundary_state'

```

Comment:
Unfortunately, `boundary_state` is missing from [the experimental dataset used for testing](https://grass.osgeo.org/sampledata/north_carolina/nc_spm_full_v2alpha2.tar.gz). Can you please use `nc_state` for now so that it runs in GitHub Actions?